CHANGELOG.rst

```diff
@@ -2,6 +2,85 @@
 SDK Changelog
 =============
 
+1.4.0 (2023-06-21)
+==================
+
+SDK for the upcoming Open Forms 2.2 release.
+
+.. warning:: SDK 1.4.0 requires at least version 2.2.0 of the backend API.
```

Comment:
```suggestion
.. warning:: SDK 1.4.0 requires at least version 2.2.0 of Open Formulieren.
```

Might be more clear since we don't have a project called "backend API".

Reply:
I'll rename it to "Open Forms API", the language here mimicks the documentation: https://open-forms.readthedocs.io/en/stable/developers/versioning.html#open-forms-api